### ISSUE TITLE
The prediction result is only <eos> when conformer decoding

### ISSUE THREAD

**Issue by oshindow** [espnet/espnet#2900]
**Describe the bug**
We used a conformer script to train an ASR model and got a 9% ERR in the test set. But when we tried a new test set, all sentence decoding result was only `<eos> `

**Error logs**
the decode.log is as follows:
(asr:1015) INFO: (1/120) decoding 20210121134207
(e2e_asr_transformer:293) INFO: input lengths: 14
(e2e_asr_transformer:309) INFO: max output length: 14
(e2e_asr_transformer:310) INFO: min output length: 0
(e2e_asr_transformer:439) INFO: end detected at 6
(e2e_asr_transformer:472) total log probability: -5.355807304382324
(e2e_asr_transformer:475) normalized log probability: -2.677903652191162
(asr_utils:852) INFO: groundtruth: 三千六百一十二点六对三千六百一十二点六元
(asr_utils:853) INFO:perdiction: `<eos>`
(asr:1015) INFO: (2/120) decoding 20210121134208
(e2e_asr_transformer:293) INFO: input lengths: 24
(e2e_asr_transformer:309) INFO: max output length: 24
(e2e_asr_transformer:310) INFO: min output length: 0
(e2e_asr_transformer:439) INFO: end detected at 6
(e2e_asr_transformer:472) total log probability: -0.566754736537761
(e2e_asr_transformer:475) normalized log probability: -2.7188236713409424
(asr_utils:852) INFO: groundtruth: 三千六百一十二点六对三千六百一十二点六元
(asr_utils:853) INFO:perdiction: `<eos>`

**Comment by okkteam** [comment]
Then we checked the data to make sure they can hear clearly with 16k sample rate and 16 bit; however, the problem remains

**Comment by kan-bayashi** [comment]
Did you perform normalization correctly?

**Comment by okkteam** [comment]
> Did you perform normalization correctly?

Yes, I think we perform normalization correctly. We add {train_set}/cmvn.ark to the new test set before training. Btw, could there be a broken while we transfer 8k,8bit data to 16k,16bit data by sox. Is there any check method? The only method we used is listening.

**Comment by okkteam** [comment]
> Did you perform normalization correctly?
Maybe we should compute the new test set cmvn.ark, and add it to itself...

**Comment by okkteam** [comment]
> Did you perform normalization correctly?

Maybe we should compute the new test set cmvn.ark, and add it to itself...

**Comment by kan-bayashi** [comment]
> Maybe we should compute the new test set cmvn.ark, and add it to itself...

You need to use `cmvn.ark` computed over training data.
Maybe you can follow the procedure:
https://github.com/espnet/espnet/blob/master/utils/recog_wav.sh

**Comment by okkteam** [comment]
> > Maybe we should compute the new test set cmvn.ark, and add it to itself...
> 
> You need to use `cmvn.ark` computed over training data.
> Maybe you can follow the procedure:
> https://github.com/espnet/espnet/blob/master/utils/recog_wav.sh

Thanks, it's a useful script. We did the steps correctly, but still no output results.
